<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>900</width>
      <height>606</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="sceneCoord">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QLabel" name="viewCoord">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QLabel" name="mapCoord">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn_Default">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>90</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>默认模式</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn_AddLoc">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>130</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>添加目标点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBtn_AddPath">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>170</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>添加路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_Save">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_Load">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>读取</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textLog">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>340</y>
      <width>261</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_Clear">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
